app/src/metrics.ts

```diff
@@ -5,7 +5,7 @@ import {
   HydratedGitCommitObject,
   HydratedGitTreeObject,
 } from "../../parser/src/model"
-import { unionAuthors } from "./util"
+import { dateFormat as dateFormatLong, unionAuthors } from "./util"
```

Comment:
I think you tried to rename this, but it instead just got renamed locally, see other comment.